src/app/landing-page/landing-page.component.html

```diff
@@ -0,0 +1,24 @@
+<p class="header-message">Request to join the
+  <br>
+  <strong>Swiftfest</strong> slack channel
+</p>
+<br>
+<p *ngIf="(emailValid===null)" class="email-message">Enter your <strong>email address</strong></p>
+<p *ngIf="(emailValid===false || captchaResponse===null)"
```

Comment:
this logic should go in the component not in the template

Reply:
Done, went into the component